test/test_broker/acceptance/service-fabrik-api-2.0.instances.director.spec.js

```diff
@@ -1980,18 +1863,12 @@ describe('service-fabrik-api-sf2.0', function () {
 
         it('should return 201 OK', function () {
           mocks.uaa.tokenKey();
-          mocks.cloudController.getServiceInstance(instance_id, {
-            service_guid: service_id,
-            space_guid: space_guid,
-            service_plan_guid: plan_id
-          });
+          mocks.apiServerEventMesh.nockGetResource(CONST.APISERVER.RESOURCE_GROUPS.DEPLOYMENT, CONST.APISERVER.RESOURCE_TYPES.DIRECTOR, instance_id, dummyDeploymentResource);
           mocks.cloudController.getSpace(space_guid, {
             organization_guid: organization_guid
           });
           mocks.cloudController.getOrganization(organization_guid);
```

Comment:
If I understand correctly, ```getSpace``` and ```getOrganization```, will not be called fro k8s, maybe you can add that test as well.

Reply:
Same comment for file: service-fabrik-api.instances.director.spec.js

Reply:
Currently end-to-end flow including auth for k8s is unclear, hence added single UT to cover the k8s branch as of now.